modules/mod_base/controllers/controller_file.erl

```diff
@@ -8,17 +8,17 @@
 %% Licensed under the Apache License, Version 2.0 (the "License");
 %% you may not use this file except in compliance with the License.
 %% You may obtain a copy of the License at
-%% 
+%%
 %%     http://www.apache.org/licenses/LICENSE-2.0
-%% 
+%%
```

Comment:
Would it be possible to strip the whitespace of all source files in one commit? Otherwise we'll be seeing these sanitary changes for a long time.